src/pybricks/_common.py

```diff
@@ -1285,3 +1285,77 @@ def hsv(self, surface: bool = True) -> Color:
             Measured color. The color is described by a hue (0--359), a
             saturation (0--100), and a brightness value (0--100).
         """
+
+
+class BLE:
+    """
+    Bluetooth Low Energy.
+
+    .. versionadded:: 3.3
+    """
+
+    def broadcast(self, *args: Union[None, bool, int, float, str, bytes]) -> None:
+        """broadcast([arg1][, arg2][...])
```

Comment:
As a beginning Python user, I could never understand this type of signature. (It's one of the reason I'm not a fan of *args.) Also note that this puts the indexing off by one. 

Suggested:

```python
broadcast(value0, value1, ...)
```